random.go

```diff
@@ -54,3 +56,68 @@ func Truncate(val float64, precision int) float64 {
 	multiplier := math.Pow(10, float64(precision))
 	return math.Floor(val*multiplier) / multiplier
 }
+
+func DeterministicRandom(seedHex string, sequenceNr int, probabilities []float64) (int, error) {
```

Comment:
@FTHans would you be able to use `int64` instead of `int`?

The reason I am asking is that when I am selecting the pointer from the database to understand at what index the wheel is, I am getting an int64 from the database. Maybe int64 at least for the `sequenceNr` at least? Probably the output `int` can remain the same.